action.yml

```diff
@@ -272,7 +272,7 @@ runs:
           format('{0}/cache', github.workspace) ||
           inputs.terraform-cache-dir }}
         echo "TF_PLUGIN_CACHE_DIR=$CACHE_DIR" >> $GITHUB_ENV
-        echo "TERRAGRUNT_PROVIDER_CACHE_DIR=$CACHE_DIR" >> $GITHUB_ENV
+        echo "TG_PROVIDER_CACHE_DIR=$CACHE_DIR" >> $GITHUB_ENV
```

Comment:
we need both TERRAGRUNT_PROVIDER_CACHE_DIR and TG_PROVIDER_CACHE_DIR here to be backwards compatible with older terragrunt versions